frontend/src/store/useTaskStore.js

```
@@ -0,0 +1,103 @@
+  isUpdatingTask: false,
+  isDeletingTask: false,
+
+  fetchTasks: async () => {
+    set({ isFetchingTasks: true, taskError: null });
+    try {
+      const response = await axiosInstance.get("/tasks");
+      set({ tasks: response.data });
+      
+    } catch (error) {
+      console.error("Error fetching tasks:", error);
+      set({ taskError: "Failed to fetch tasks: " + error.message });
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)